src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const threeDayDiscount = 20;
+  const sevenDayDiscount = 50;
+  const basePrice = 40;
+
+  if (days >= 7) {
+    return days * basePrice - sevenDayDiscount;
+  }
+
+  if (days >= 3) {
```

Comment:
Similarly, the condition `days >= 3` should not use a hardcoded literal. It's better to define a constant like `const SHORT_TERM = 3;` and use it in the condition, as per the checklist item 2.